topi/python/topi/arm_cpu/conv2d.py

```diff
@@ -523,9 +523,8 @@ def _callback(op):
 
 ##### REGISTER ALTER OP LAYOUT #####
 @conv2d_alter_layout.register(["arm_cpu"])
-def _alter_conv2d_layout_arm(attrs, inputs, tinfos):
+def _alter_conv2d_layout_arm(attrs, inputs, tinfos, F):
```

Comment:
Could we provide a documentation comment here with information about the parameters including F?